page-home.php

```diff
@@ -7,25 +7,104 @@
 
 <?php get_template_part( 'breadcrumb' ); ?>
 
-<div id="primary" class="content-area latin-home">
+<div id="primary" class="content-area latin-home level-one">
 	<div class="container">
-		<div class="row">
-			<main id="main" class="col-xs-12 col-sm-8" role="main">
+        <div class="row" role="banner">
+            <div class="col-md-12">
 				<?php
-				while ( have_posts() ) : the_post();
-					get_template_part( 'content' );
-				endwhile;
+				global $post;
+				$image      = make_path_relative_no_pre_path( get_feature_image_url( $post->ID, 'full-page-width', true ) );
+				$title      = get_the_title();
+				get_page_banner( 'level one', $title, $image, '', '', '' );
 				?>
-			</main>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+                <article>
+                    <div class="entry-header">
+                        <h2><?php
+	                        echo get_post_meta($post -> ID, 'latin_h2', true);
+	                        ?></h2>
+                    </div>
+                    <div class="row entry-content">
+                        <div class="col-xs-12 col-sm-8 col-md-8">
+                            <?php
+                            while ( have_posts() ) : the_post();
+	                           the_content();
+
+                            endwhile;
+                            ?>
+                        </div>
+                    </div>
+                </article>
+            </div>
+        </div>
+        <div class="row equal-heights" id="equal-heights">
```

Comment:
Equal height doesn't appear to be working. Please check it's correctly configured.

Reply:
@AshTNA we need to add similar code to the theme https://github.com/nationalarchives/tna-base/blob/develop/inc/functions/tna-functions.php#L20

Reply:
Hi Simon, this has been done